feature-libs/checkout/b2b/styles/components/steps/type/_payment-type.scss

```diff
@@ -0,0 +1,22 @@
+cx-payment-type {
```

Comment:
Not sure how much of a problem is to rename the `type` folder to `payment-type`?

Reply:
no problem, kept it for consistency from old checkout, but I don't mind changing